frontend/src/Pages/Students.tsx

```diff
@@ -0,0 +1,340 @@
+    );
+    const userData = data?.data as User[] | [];
+    const showToast = (message: string, state: ToastState) => {
+        setToast({
+            state,
+            message,
+            reset: () => {
+                setToast({
+                    state: ToastState.success,
+                    message: '',
+                    reset: () => {
+                        setDisplayToast(false);
+                    }
+                });
+            }
+        });
+        setDisplayToast(true);
+    };
+
+    function resetModal() {
+        setTimeout(() => {
+            setTargetUser(null);
+        }, 200);
+    }
+
+    const deleteUser = async () => {
+        if (targetUser?.id === DEFAULT_ADMIN_ID) {
+            showToast(
+                'This is the primary administrator and cannot be deleted',
+                ToastState.error
+            );
+            return;
+        }
+        const response = await API.delete('users/' + targetUser?.id);
+        const toastType = response.success
+            ? ToastState.success
+            : ToastState.error;
+        const message = response.success
+            ? 'User deleted successfully'
+            : response.statusText;
+        deleteUserModal.current?.close();
+        showToast(message, toastType);
+        resetModal();
+        mutate();
+        return;
+    };
+
+    const onAddUserSuccess = (pswd = '', msg: string, type: ToastState) => {
+        showToast(msg, type);
+        setTempPassword(pswd);
+        addUserModal.current?.close();
+        showUserPassword.current?.showModal();
+        mutate();
+    };
+
+    const hanldleEditUser = () => {
+        editUserModal.current?.close();
+        resetModal();
+        mutate();
+    };
+
+    const handleDeleteUserCancel = () => {
+        deleteUserModal.current?.close();
+        resetModal();
+    };
+
+    const handleResetPasswordCancel = (msg: string, err: boolean) => {
+        const state = err ? ToastState.error : ToastState.success;
+        if (msg === '' && !err) {
+            resetUserPasswordModal.current?.close();
+            resetModal();
+            return;
+        }
+        showToast(msg, state);
+        resetModal();
+    };
+
+    const handleDisplayTempPassword = (psw: string) => {
+        setTempPassword(psw);
+        resetUserPasswordModal.current?.close();
+        showUserPassword.current?.showModal();
+        showToast('Password Successfully Reset', ToastState.success);
+    };
+
+    const handleShowPasswordClose = () => {
+        showUserPassword.current?.close();
+        setTempPassword('');
+        resetModal();
+    };
+
+    const handleChange = (newSearch: string) => {
+        setSearchTerm(newSearch);
+        setPageQuery(1);
+    };
+
+    return (
+        <AuthenticatedLayout title="Students" path={['Users']}>
+            <div className="flex flex-col space-y-6 overflow-x-auto rounded-lg p-4">
```

Comment:
Would you be able to add in a header here that specifies this is the student management page? Like it shows in the screenshot below
<img width="928" alt="Screenshot 2024-09-25 at 12 36 13 PM" src="https://github.com/user-attachments/assets/a753882d-f57a-4280-9fee-cffbff543760">
Should be a quick fix- just adding in `<h1>Student Management</h1>` in there with